pages/index.vue

```diff
@@ -34,39 +266,294 @@ export default {
 </script>
 
 <style>
-/* Sample `apply` at-rules with Tailwind CSS
-.container {
-  @apply min-h-screen flex justify-center items-center text-center mx-auto;
+body {
```

Comment:
I was planning to use `tailwindcss` for everything in order to avoid writing CSS and help keep the styling consistent.
I guess this is fine for a first draft, but in the future we will most likely drop it.

Reply:
Fair point, I'm not familiar with Tailwind and just wanted to get something moving, I've reviewed it and can see a couple of ways that I could integrate it, I will experiments on another branch